presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloConnector.java

```diff
@@ -94,9 +96,13 @@ public ConnectorTransactionHandle beginTransaction(IsolationLevel isolationLevel
     }
 
     @Override
-    public void commit(ConnectorTransactionHandle transactionHandle)
+    public Optional<ConnectorCommitResult> commit(ConnectorTransactionHandle transactionHandle)
     {
-        checkArgument(transactions.remove(transactionHandle) != null, "no such transaction: %s", transactionHandle);
+        checkArgument(
```

Comment:
Was this change applied by some tools automatically? The PR is already very large. It would be good to avoid making this kind of pure format changes. This will make review much easier.